File: iOS/Code/Xamarin/bugTrap/bugTrapKit/Controllers/NavigationControllers/BtImageNavigationController.cs


using System;

using Foundation;
using UIKit;

namespace bugTrapKit
{
	public partial class BtImageNavigationController : BtNavigationController
	{
		public bool SelectingSnapshotImagesForExport { get; set; }

		public BtImageNavigationController (IntPtr handle) : base(handle)
		{
		}
	}
}

File: iOS/Code/Swift/bugTrap/bugTrapKit/Controllers/NavigationControllers/BtImageNavigationController.swift
//
//  BtImageNavigationController.swift
//  bugTrap
//
//  Created by Colby L Williams on 10/28/14.
//  Copyright (c) 2014 bugTrap. All rights reserved.
//

import UIKit

class BtImageNavigationController: BtNavigationController {

	// class let storyboardId = "BtImageNavigationController"
	
	var selectingSnapshotImagesForExport = false
	
	override func prefersStatusBarHidden() -> Bool {
		return true
	}
	
	override func preferredStatusBarStyle() -> UIStatusBarStyle {
		return .LightContent
	}
}
